setup.py

```diff
@@ -32,8 +32,11 @@
     classifiers=[
         'License :: OSI Approved :: BSD License',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 2.6',
         'Programming Language :: Python :: 3',
-    ]
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
```

Comment:
I generally skip these specific classifiers in favour of the `python_requires` field, because the classifiers just get outdated. But I don't feel strongly about it.